samples/dialogs/src/main/java/com/badoo/ribs/samples/dialogs/rib/dialogs_example/DialogsView.kt

```diff
@@ -9,17 +9,14 @@ import com.badoo.ribs.core.view.AndroidRibView
 import com.badoo.ribs.core.view.RibView
 import com.badoo.ribs.core.view.ViewFactory
 import com.badoo.ribs.samples.dialogs.R
-import com.badoo.ribs.samples.dialogs.rib.dialogs_example.DialogsView.Event
 import com.badoo.ribs.samples.dialogs.rib.dialogs_example.DialogsView.Event.ShowThemedDialogClicked
 import com.badoo.ribs.samples.dialogs.rib.dialogs_example.DialogsView.Event.ShowSimpleDialogClicked
 import com.badoo.ribs.samples.dialogs.rib.dialogs_example.DialogsView.Event.ShowLazyDialogClicked
 import com.badoo.ribs.samples.dialogs.rib.dialogs_example.DialogsView.Event.ShowRibDialogClicked
 import com.badoo.ribs.samples.dialogs.rib.dialogs_example.DialogsView.ViewModel
-import com.jakewharton.rxrelay2.PublishRelay
-import io.reactivex.ObservableSource
 import io.reactivex.functions.Consumer
 
-interface DialogsView : RibView, ObservableSource<Event>, Consumer<ViewModel> {
+interface DialogsView : RibView, Consumer<ViewModel> {
```

Comment:
```suggestion
interface DialogsView : RibView {
```

Reply:
This is actually needed for calling `view.accept()` in the presenter, because we pass the View interface to the presenter, not the implementation.